test/honeysql/core_test.clj

```diff
@@ -15,6 +15,7 @@
                (join :draq [:= :f.b :draq.x])
                (left-join [:clod :c] [:= :f.a :c.d])
                (right-join :bock [:= :bock.z :c.e])
+               (full-join :beck [:= :beck.x :c.y])
```

Comment:
Yeah, that's a fair point, maybe we merge this now and circle back to testing the merge helpers.